theopensystemslab/planx-new
# feat: add new component type for sections

![Screenshot from 2023-02-21 08-27-39](https://user-images.githubusercontent.com/5132349/220277894-fa53ba06-af46-489f-867b-3d154ad7a302.png)

Stubs out a basic "Section" component !

To test: 
- The nodeId is added to your breadcrumbs as you navigate through a flow, we can cross reference the flow schema to get the "type" fo this nodeId
- The passport updates `"currentSection"` property when it passes a section node - the passport isn't ultimately a sensible place to track this so totally expect this will change and be replaced by Navigation store methods in the near future, but gives us something simple to work with immediately! 
- Sections are not visible from preview environments, they are always recorded as `"auto": true` in the breadcrumbs
